utils/intentmetrics.py

```diff
@@ -60,6 +60,10 @@ def func(args):
 
             precisions[idx] = precision
             recalls[idx] = recall
+            print ("precision")
+            print (precision)
+            print ("recall")
+            print (recall)
```

Comment:
These trace statements should not be printed

Reply:
doh, that wasn't meant to go in.  I'll get rid of it.

Reply:
Reverted change.